packages/main/cypress/specs/BusyIndicator.cy.tsx

```diff
@@ -24,3 +29,186 @@ describe("Rendering", () => {
 			.should("exist");
 	});
 });
+
+describe("BusyIndicator general interaction", () => {
+	it("tests event propagation", () => {
+		const onClickStub = cy.stub().as("myStub");;
+		cy.mount(
+			<BusyIndicator id="busy-tree" class="busyindicator8auto">
+				<Tree id="treeDynamic">
+					<TreeItem id="treeItem" text="Has preloaded children" onClick={onClickStub}>
+						<TreeItem text="Child 1"></TreeItem>
+						<TreeItem text="Child 2"></TreeItem>
+					</TreeItem>
+				</Tree>
+			</BusyIndicator>
+		);
+
+		cy.get("#treeItem")
+			.shadow()
+			.find("ui5-icon.ui5-li-tree-toggle-icon")
+			.realClick();
+
+		cy.realPress("Space");
+
+		cy.get("@myStub").should("not.have.been.called");
+	});
+
+	it("test activation", () => {
+		cy.mount(
+			<BusyIndicator id="busy-container" class="busyindicator4auto" size="M" text="Loading">
+				<List id="fetch-list" no-data-text="No Data" header-text="Available Items" class="busyindicator5auto"></List>
+			</BusyIndicator>
+		);
+
+		cy.get("#busy-container")
+			.shadow()
+			.find(".ui5-busy-indicator-busy-area")
+			.should("not.exist");
+
+		cy.get("#busy-container")
+			.invoke("attr", "active", "");
+
+		cy.wait(3000);
+
+		cy.get("#busy-container")
+			.shadow()
+			.find(".ui5-busy-indicator-busy-area")
+			.should("exist");
+
+		cy.get("#busy-container")
+			.invoke("removeAttr", "active");
+
+		cy.get("#busy-container")
+			.shadow()
+			.find(".ui5-busy-indicator-busy-area")
+			.should("not.exist");
+	});
+
+	it("tests focus handling", () => {
+		cy.mount(<BusyIndicator size="M" active id="indicator1"></BusyIndicator>);
+
+		cy.get("#indicator1").realClick();
+
+		cy.get("#indicator1")
+			.shadow()
+			.find(".ui5-busy-indicator-busy-area")
+			.should("exist");
+	});
+
+	it("tests internal focused element attributes", () => {
+		cy.mount(<BusyIndicator size="M" active id="indicator1"></BusyIndicator>);
+
+		cy.get("#indicator1").realClick();
+
+		cy.get("#indicator1")
+			.shadow()
+			.find(".ui5-busy-indicator-busy-area")
+			.should("have.attr", "role", "progressbar")
+			.and("have.attr", "tabindex", "0")
+			.and("have.attr", "aria-valuemin", "0")
+			.and("have.attr", "aria-valuemax", "100")
+			.and("have.attr", "aria-valuetext", "Busy");
+	});
+
+	it("tests content is not reachable with keyboard when active in both directions", () => {
+		cy.mount(
+			<div id="test">
+				<Button id="beforeIndicatorWithBtn">focus stop before</Button>
+				<BusyIndicator id="indicatorWithBtn" size="M" active>
+					<Button>Hello World</Button>
+				</BusyIndicator>
+				<Button id="afterIndicatorWithBtn" >focus stop after</Button>
+			</div>
+		);
+
+		cy.get("#beforeIndicatorWithBtn").realClick();
+		cy.realPress("Tab");
+
+		cy.get("#test")
+			.find("[active]")
+			.should("have.id", "indicatorWithBtn");
```

Comment:
Remove this. Rather add assertions that check if specific element in the busy indicator's shadow root is focused and the "Hello World" button is not focused.